table/tables/mutation_checker.go

```diff
@@ -90,14 +90,6 @@ func CheckDataConsistency(
 
 	columnMaps := getColumnMaps(txn, t)
 
-	if rowToInsert != nil {
```

Comment:
Seems this is not "disable" but  "remove". Maybe we could change this part to comments first and enable it after refactoring?